### PR TITLE
*: lint code with github actions

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

- lint code with GitHub actions

### What is changed and how it works?

- fix issues reported by lint tools
- refine some test cases after fixed issues

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: pkg/atomic2/atomic_test.go [hunk 1]
```diff
@@ -40,7 +40,6 @@ func (t *testAtomicSuite) TestAtomicError(c *check.C) {
 	err2 = e.Get()
 	c.Assert(err2, check.DeepEquals, err)
 
-	err = errors.New("test2")
```
Comment (lance6716) [inline pkg/atomic2/atomic_test.go:43]: I copied from chunzhu's code, guess he want to check `e.Set(err)` doesn't get a reference to `err`

**Review by lance6716 — COMMENTED** [review]
rest LGTM

File: pkg/atomic2/atomic_test.go (hunk 1 above)
Comment (csuzhangxc) [reply pkg/atomic2/atomic_test.go:43]: I added it back and a check with it.

**Comment by lance6716** [comment]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**